src/pages/Barrels/PoolCard.tsx

```diff
@@ -148,6 +164,26 @@ function PoolCard({ token, hToken, bonusToken }: PoolCardProps) {
           <EntryTitle>Total Reward:</EntryTitle>
           <TokenAmountWithoutIcon symbol={token.symbol} amount={hToken.totalReward} decimals={token.decimals} />
         </Entry>
+        {userMaxEstimatedReward.gt(0) && (
```

Comment:
also whatever we decide to do, It's better to just show it anyway even if it's 0, so all cards have the same height